### DISCUSSION TITLE
Retri Hekili

### DISCUSSION THREAD

**Discussion by Rafa28062013** [Hekili/hekili#937] [Q&A] [answered]
What is this button for? Check Wake of Ashes Range? When to use this button.

See Link : https://ibb.co/nrn13JD

**Comment by Hekili** [comment, marked as answer]
If checked, Wake of Ashes is not recommended while you're out of range of your target.  See /hekili > Retribution for any additional info.

**Reply by Rafa28062013** [reply]
Should I turn this on or leave it off?

**Reply by Hekili** [reply]
It's a preference -- use the information in its description to decide whether you want it on or off.
